3D-microservices-observability/springboot-jersey-app-2/README.md

```diff
@@ -0,0 +1,78 @@
+# Wavefront Distributed Tracing Sample Apps - Springboot Jersey App 2
+
+This is a sample java application using Springboot beachshirts (#[beachops](https://medium.com/@matthewzeier/thoughts-from-an-operations-wrangler-how-we-use-alerts-to-monitor-wavefront-71329c5e57a8)) 
+which makes shirts for the beach. The application is based on spring boot web using rest templates and rest annotations.
+- JDK 1.8 and up
+- Spring Boot 2.3.0.RC1
```

Comment:
Can this be updated (and the pom.xml) to a non-RC version? It looks like `2.3.0.RELEASE` was released May 2020.